qiskit_experiments/library/driven_freq_tuning/coefficients.py

```diff
@@ -181,6 +181,7 @@ def __eq__(self, other):
                 self.neg_coef_o1 == other.neg_coef_o1,
                 self.neg_coef_o2 == other.neg_coef_o2,
                 self.neg_coef_o3 == other.neg_coef_o3,
+                self.offset == other.offset,
```

Comment:
This is a followup of #1236 . PR can be separated.

Reply:
One of the nice things about dataclasses :slightly_smiling_face: